recipes/azure-datalake-store/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "azure-datalake-store" %}
+{% set version = "0.0.47" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 55bcbec99a35a52ea291518dadfd94c3b645e66d05ce927228289b93979ad3a5
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```

Could you please change it to
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```